frontend/packages/dev-console/src/components/import/serverless-function/AddServerlessFunctionForm.tsx

```diff
@@ -61,6 +72,8 @@ const AddServerlessFunctionForm: React.FC<
     image,
   } = values;
   const isPipelineEnabled = useFlag(FLAG_OPENSHIFT_PIPELINE);
+  const imgSrcVSCode = getImageForIconClass('icon-vscode');
+  const imgScrIntelliJ = getImageForIconClass('icon-intellij');
```

Comment:
You can import logos also like this:

```ts
import * as vscodeIcon from './icon-vscode.png';
```

and use it like this:

```ts
<img src={vscodeIcon} 
```

The `getImageForIconClass` is to get dynamic images.